.cspell.project-words.txt

```diff
@@ -40,3 +40,8 @@ adsb
 oneshot
 thiserror
 Refactorings
+schipol
```

Comment:
missing 'h', might recognize it if you fix the typo in the code

Reply:
https://github.com/Arrow-air/svc-compliance/pull/16/commits/f4d6fe8cd6bd04e5872a2b94a5f6e3b73ababb50